hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/service/model/Question.java

```diff
@@ -27,8 +27,10 @@
 package org.alfresco.hxi_connector.hxi_extension.service.model;
 
 import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 
+@Getter
```

Comment:
This is an internal service model, maybe we should use here `@Data` instead of `@Getter` and `@RequiredArgsConstructor`? 🤔

Reply:
done